jobs/build/build-sync/Jenkinsfile

```diff
@@ -37,6 +37,8 @@ node() {
                 commonlib.mockParam(),
                 commonlib.ocpVersionParam('BUILD_VERSION', '4'),
                 commonlib.artToolsParam(),
+                commonlib.enableTelemetryParam(),
+                commonlib.telemetryEndpointParam(),
```

Comment:
We're not using the `OTEL_EXPORTER_OTLP_ENDPOINT`, right? If not blank, we should probably pass it on to build-sync so we can override

Reply:
If we set it there is an env var with the same name, build-sync will pick it up then

Reply:
@lgarciaaco I would prefer to make it explicit to set the env var for OTEL_EXPORTER_OTLP_ENDPOINT. WDYT?